### PR TITLE
Do not dnf install docs and multiple languages

### PR DESCRIPTION
**Pull request by mykaul**
### Explain the changes
1. Reduce slightly the container size by not installing docs and multiple languages.

### CODE REVIEW

**Comment by nimrod-becker** [comment]
Can you please git squash?

**Review by liranmauda — DISMISSED** [review]

File: src/deploy/NVA_build/NooBaa.Dockerfile [hunk 1]
```diff
@@ -33,6 +33,14 @@ RUN tar \
...
 #####################################################################################################################################
 
+# Do not install docs and multiple langaugages.
+LANG="en_US"
+echo "%_install_lang $LANG" > /etc/rpm/macros.image-language-conf
+
+awk '(NF==0&&!done){print "override_install_langs='$LANG'\ntsflags=nodocs";done=1}{print}' \
+    < /etc/dnf/dnf.conf > /etc/dnf/dnf.conf.new
+mv /etc/dnf/dnf.conf.new /etc/dnf/dnf.conf
+
```
Comment (liranmauda) [inline src/deploy/NVA_build/NooBaa.Dockerfile]: should move to line 51: https://github.com/noobaa/noobaa-core/pull/5857/files#diff-e260bb3018701cb64c7b23e88c7fc672R51

In order for the changes to affect it should be on the last stage.

**Review by liranmauda — CHANGES_REQUESTED** [review]

File: src/deploy/NVA_build/NooBaa.Dockerfile (hunk 1 above)
Comment (liranmauda) [reply src/deploy/NVA_build/NooBaa.Dockerfile]: should be between:

FROM centos:8

ENV container docker

File: src/deploy/NVA_build/NooBaa.Dockerfile [hunk 2]
```diff
@@ -33,13 +33,21 @@ RUN tar \
...
+FROM centos:8
+
 ##############################################################
 #   Title: Start of the Server Image
 #   Size: ~ 841 MB
 #   Cache: Rebuild when any layer is changing
 ##############################################################
 
-FROM centos:8
+# Do not install docs and multiple langaugages.
```
Comment (mykaul) [inline src/deploy/NVA_build/NooBaa.Dockerfile]: Ah, typo. ARGH!

File: src/deploy/NVA_build/NooBaa.Dockerfile [hunk 3]
```diff
@@ -33,13 +33,21 @@ RUN tar \
...
 ##############################################################
 
-FROM centos:8
+# Do not install docs and multiple languages.
+LANG="en_US"
+echo "%_install_lang $LANG" > /etc/rpm/macros.image-language-conf
+
+awk '(NF==0&&!done){print "override_install_langs='$LANG'\ntsflags=nodocs";done=1}{print}' \
+    < /etc/dnf/dnf.conf > /etc/dnf/dnf.conf.new
+mv /etc/dnf/dnf.conf.new /etc/dnf/dnf.conf
```
Comment (liranmauda) [inline src/deploy/NVA_build/NooBaa.Dockerfile]: This should be a DockerFile Syntax ( missed that on the first look :) )
```

# Do not install docs and multiple languages.
RUN LANG="en_US" && \
    echo "%_install_lang $LANG" > /etc/rpm/macros.image-language-conf && \
    awk '(NF==0&&!done){print "override_install_langs='$LANG'\ntsflags=nodocs";done=1}{print}' < /etc/dnf/dnf.conf > /etc/dnf/dnf.conf.new && \
    mv /etc/dnf/dnf.conf.new /etc/dnf/dnf.conf
```

**Review by liranmauda — CHANGES_REQUESTED** [review]

**Comment by nimrod-becker** [comment]
@mykaul Building with and without the change results in the same image size, as well as the same hash...

**Comment by mykaul** [comment]
> @mykaul Building with and without the change results in the same image size, as well as the same hash...
> (at least US)

So either I made a mistake, or it's time I'll stop being lazy and build locally...

**Comment by mykaul** [comment]
OK, sent other PRs to improve the Makefile. We'll re-visit this once they are done with, as currently I can't even build locally.

**Pull request closed without merging**